corehq/apps/locations/management/commands/delete_locations.py

```diff
@@ -0,0 +1,51 @@
+from django.core.management.base import BaseCommand
+from dimagi.utils.couch.database import iter_docs
+from corehq.apps.domain.models import Domain
+from corehq.apps.locations.models import Location
+from .check_loc_types import locs_by_domain
+
+
+class Command(BaseCommand):
+    args = "<domain>"
+    help = ("Change the doc types of all locations in `domain` to "
+            "Location-DELETED.  Note that this only affects couch locations.")
+
+    def bulk_delete_locs(self, loc_ids, total):
+        locs_to_save = []
+        count = 0
+        for loc in iter_docs(Location.get_db(), loc_ids):
+            loc['doc_type'] = "{}-DELETED".format(loc['doc_type'])
```

Comment:
Isn't the convention "-Deleted". Don't know if this will make a difference or not, but I see a couple references to location docs being set as "-Deleted". Could use this constant maybe? https://github.com/dimagi/dimagi-utils/blob/master/dimagi/utils/couch/undo.py#L4